lbrynet/lbrynet_daemon/DaemonControl.py

```diff
@@ -115,7 +115,7 @@ def update_settings_from_args(args):
     if args.branch:
         to_pass['ui_branch'] = args.branch
     to_pass['use_auth_http'] = args.useauth
-    to_pass['wallet_type'] = args.wallet
+    to_pass['wallet'] = args.wallet
```

Comment:
worth considering, in the future, adding a check in `settings.update()` to require that the setting already exists, would catch errors like this.